src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
+});
+
+const thirdPromise = new Promise((resolve, reject) => {
+  let leftClick = 0;
+  let rightClick = 0;
```

Comment:
I'd rather use a boolean value instead of a number, but not critical here (just FYI)